_positions/product-manager.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Product Manager
+permalink: positions/product-manager/
+team: 18F
+active: true
+---
+##Basic Information:
+**Title:** Product Manager  <br>
+**Series & Grade Level:** GS-2210-14/15 <br>
+**Salary Range:** Base Salary GS-14 - $87,263 to $113,444; Base Salary GS-15 - $102,646 to $133,444. Total compensation will include locality pay based on the individual's duty location. For locality pay details, please visit: https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/ <br>
+**Work Schedule:** Full Time<br>
+
+**Job Announcement Number:** 17001-Product Manager<br>
+**Open:** DATE<br>
```

Comment:
Thanks for noting. I hate to suggest further delay, but if it's five calendar days inclusive of the open date, might we get more applicants if we waited until after the holidays to post this?